src/precompile.jl

```diff
@@ -1,15 +1,13 @@
-using SnoopPrecompile
+using PrecompileTools
 
-@info "SnoopPrecompile is analyzing PlanetaryEphemeris.jl code..."
-
-@precompile_setup begin
+@setup_workload begin
     # Starting time of integration
     jd0 = datetime2julian(DateTime(2000,1,1,12)) 
     # Number of years 
     nyears = 2031.0 - year(julian2datetime(jd0))
-
-    @precompile_all_calls begin
-        propagate(1, jd0, nyears, Val(false); dynamics = DE430!, order = 25, abstol = 1.0E-20)
+    @compile_workload begin
+        # All calls in this block will be precompiled, regardless of whether
+        # they belong to your package or not (on Julia 1.8 and higher)
         propagate(1, jd0, nyears, Val(true); dynamics = DE430!, order = 25, abstol = 1.0E-20)
```

Comment:
Does it change anything to use `propagate` for a two-step integration?

Reply:
Integrating two steps changes precompilation time by a few minutes. I've noted that the actual integration represents only a small percentage of precompilation time. This PR is a solution to https://github.com/PerezHz/PlanetaryEphemeris.jl/issues/13, but we should investigate further why the package takes so long precompilating. If you agree I will bump the patch version and merge...